packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin.tsx

```diff
@@ -230,6 +235,10 @@ export const TooltipPlugin: React.FC<TooltipPluginProps> = ({
         });
       }
 
+      if (sortOrder !== SortOrder.None) {
+        series.sort((a, b) => arrayUtils.sortValues(sortOrder)(a.value, b.value));
```

Comment:
In this, `a.value`and`b.value` is formatted display value,  whether should use the original value to sort ? I am not clearly on the context, hope someone can explain it if i wrong :)

Reply:
for example this: 
![8xbmBITuvb](https://user-images.githubusercontent.com/23014093/153582027-644b0320-8a14-417c-80f3-c7aef99e2593.png)

Reply:
Can you please open a separate issue with your findings?

Reply:
sure :)